physics.py

```diff
@@ -66,10 +74,57 @@ def __init__(self, physics_controller: PhysicsInterface, robot: MyRobot):
             for module in robot.chassis.modules
         ]
 
-        self.imu = wpilib.simulation.SimDeviceSim("navX-Sensor", 4)
+        self.arm = robot.arm
```

Comment:
Please don't keep robot objects alive longer than the robot. Grab only the things needed instead, i.e. simulation objects.

Reply:
is grabbing things like motors and solenoids to .get their output ok?

Reply:
No. Those should be available through the simulation backend as well.